news_items/041_element_question_writing/index.html

```diff
@@ -0,0 +1,172 @@
+<hr>
+<h3>New Element Features</h3>
+<br>
+<div class="card">
+  <h4 class="card-header">pl-python-variable</h4>
+  <div class="card-body">
+    <p>As mentioned above, the <code class="user-output">pl-python-variable</code> element
+      has had DataFrame display functionality deprecated in favor of the
+      <code class="user-output">pl-dataframe</code> element. However, the
+      <code class="user-output">pl-python-variable</code> element has received support for
+      displaying complex objects by way of the Python
+      <a href="https://docs.python.org/3/library/pprint.html">pprint</a> module. This
+      makes it much easier to print complex dictionaries and other nested data types.
+    </p>
+
+    <img class="border rounded" src="pl-python-variable.png" width="800px">
+
+    <p>For more information and examples, see the
+      <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-python-variable-element">pl-python-variable
+        documentation</a>.
+    </p>
+  </div>
+</div>
+<br>
+<div class="card">
+  <h4 class="card-header">pl-external-grader-variables</h4>
+  <div class="card-body">
+    <p> Previously, available variables for externally graded questions in Python had to be
+      defined in a <code>server.py</code> file. Now, the
+      <code class="user-output">pl-external-grader-variables</code> element has support
+      for defining these variables directly in the <code>question.html</code> file.
+      This means that <code>server.py</code> files defining just these variables can be
+      removed, thus decreasing the maintenance burden for courses.
+    </p>
+
+    <p>For more information and examples, see the
+      <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-external-grader-variables-element">pl-external-grader-variables
+        documentation</a>.
+    </p>
+  </div>
+</div>
+<br>
+<div class="card">
+  <h4 class="card-header">pl-graph</h4>
+  <div class="card-body">
+    <p> The <code class="user-output">pl-graph</code> element has expanded functionality for displaying
+      adjacency matrices (being able to handle negative weights and undirected graphs), and now has
+      support for displaying <a href="https://networkx.org/">NetworkX</a> graphs.
+    </p>
+
+    <img class="border rounded" src="pl-graph.png" width="800px">
+
+    <p>For more information and examples, see the
+      <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-graph-element">pl-graph documentation</a>.
+    </p>
+  </div>
+</div>
+<hr>
+<h3>Utility Code Improvements</h3>
+<p>
+  There have been substantial improvements to utility code available to instructors in the
+  <code>prairielearn.py</code> Python library, including greater support for JSON serialization
+  and more utility functions usable in custom questions. These changes have primarily been to facilitate writing
+  custom questions and support the new elements and features discussed above. This has all been detailed in
```

Comment:
I see the phrase "custom question" used throughout this news items. That's not a phrase we use elsewhere; what are you specifically referring to there? If I'm understanding correctly, you should just use "question".